pkg/commands/logging/s3/create.go

```diff
@@ -216,6 +214,30 @@ func (c *CreateCommand) ConstructInput(serviceID string, serviceVersion int) (*f
 	return &input, nil
 }
 
+func ValidateRedundancy(val string) (redundancy fastly.S3Redundancy, err error) {
+	switch val {
+	case string(fastly.S3RedundancyStandard):
+		redundancy = fastly.S3RedundancyStandard
+	case string(fastly.S3RedundancyIntelligentTiering):
+		redundancy = fastly.S3RedundancyIntelligentTiering
+	case string(fastly.S3RedundancyStandardIA):
+		redundancy = fastly.S3RedundancyStandardIA
+	case string(fastly.S3RedundancyOneZoneIA):
+		redundancy = fastly.S3RedundancyOneZoneIA
+	case string(fastly.S3RedundancyGlacierInstantRetrieval):
+		redundancy = fastly.S3RedundancyGlacierInstantRetrieval
+	case string(fastly.S3RedundancyGlacierFlexibleRetrieval):
+		redundancy = fastly.S3RedundancyGlacierFlexibleRetrieval
+	case string(fastly.S3RedundancyGlacierDeepArchive):
+		redundancy = fastly.S3RedundancyGlacierDeepArchive
+	case string(fastly.S3RedundancyReduced):
+		redundancy = fastly.S3RedundancyReduced
+	default:
+		err = fmt.Errorf("Unknown redundancy: " + val)
```

Comment:
Error messages in go should be lowercase. 

```suggestion
		err = fmt.Errorf("unknown redundancy: " + val)
```

Reply:
Added in https://github.com/fastly/cli/pull/641/commits/7ad861493108308ff40c5d3cc4b5f6ac59c0c536